### PR TITLE
+ sstimap initial commit

### PR DESCRIPTION
**Pull request by dualfade**
please review.
Thanks.

### CODE REVIEW

File: packages/sstimap/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+makedepends=('git' 'python-setuptools' 'python-pip')
+source=("git+https://github.com/vladko312/$_pkgname.git")
+sha512sums=('SKIP')
+install="$pkgname.install"
+
+pkgver() {
+  cd $_pkgname
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
```
Comment (D3vil0p3r) [inline packages/sstimap/PKGBUILD:23]:
```suggestion
pkgver() {
  cd $_pkgbase

  ( set -o pipefail
    git describe --long --tags --abbrev=7 2>/dev/null |
      sed 's/\([^-]*-g\)/r\1/;s/-/./g' ||
    printf "%s.%s" "$(git rev-list --count HEAD)" \
      "$(git rev-parse --short=7 HEAD)"
  )
}
```

File: packages/sstimap/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+pkgname=sstimap
+_pkgname=SSTImap
+pkgver=12.0d45b44
+pkgrel=1
+pkgdesc='Automatic SSTI detection tool with interactive interface.'
+arch=('any')
+groups=('blackarch' 'blackarch-webapp' 'blackarch-exploitation')
+url='URL pointing to download section of package'
+license=('gpl3')
```
Comment (D3vil0p3r) [inline packages/sstimap/PKGBUILD:12]:
```suggestion
license=('GPL-3.0')
```

File: packages/sstimap/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=sstimap
+_pkgname=SSTImap
+pkgver=12.0d45b44
+pkgrel=1
+pkgdesc='Automatic SSTI detection tool with interactive interface.'
+arch=('any')
+groups=('blackarch' 'blackarch-webapp' 'blackarch-exploitation')
```
Comment (D3vil0p3r) [inline packages/sstimap/PKGBUILD:10]:
```suggestion
groups=('blackarch' 'blackarch-exploitation' 'blackarch-webapp')
```

File: packages/sstimap/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+url='URL pointing to download section of package'
```
Comment (D3vil0p3r) [inline packages/sstimap/PKGBUILD:11]:
```suggestion
url='https://github.com/vladko312/SSTImap'
```

Comment (dualfade) [reply packages/sstimap/PKGBUILD:11]: my bad. fixed.

File: packages/sstimap/PKGBUILD (hunk 3 above)
Comment (dualfade) [reply packages/sstimap/PKGBUILD:10]: re ordered.

File: packages/sstimap/PKGBUILD (hunk 2 above)
Comment (dualfade) [reply packages/sstimap/PKGBUILD:12]: forgot to fix this. fixed.

File: packages/sstimap/PKGBUILD (hunk 1 above)
Comment (dualfade) [reply packages/sstimap/PKGBUILD:23]: Is this something new we are doing it now ?  This is not in the pkgbuild scripts. let me know. Thanks.

Comment (D3vil0p3r) [reply packages/sstimap/PKGBUILD:23]: [Here](https://github.com/BlackArch/blackarch-pkgbuilds/blob/b8fd779ec987a8d29f543a5cc5b729074ece638a/PKGBUILD-generic-git#L17) the reference. I think it should be extended to all PKGBUILD templates dealing with git: https://github.com/BlackArch/blackarch-pkgbuilds/pull/24

**Review by noraj — CHANGES_REQUESTED** [review]

File: packages/sstimap/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+_pkgname=SSTImap
+pkgver=v1.2.r3.g0d45b44
+pkgrel=1
+pkgdesc='Automatic SSTI detection tool with interactive interface.'
+arch=('any')
+groups=('blackarch' 'blackarch-exploitation' 'blackarch-webapp')
+url='https://github.com/vladko312/SSTImap'
+license=('GPL-3.0')
+depends=('python')
+makedepends=('git' 'python-setuptools' 'python-pip')
```
[the remark this answers is not in the document]
Comment (dualfade) [reply packages/sstimap/PKGBUILD:14]: Will remove.

File: packages/sstimap/PKGBUILD [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+pkgname=sstimap
+_pkgname=SSTImap
+pkgver=v1.2.r3.g0d45b44
+pkgrel=1
+pkgdesc='Automatic SSTI detection tool with interactive interface.'
+arch=('any')
+groups=('blackarch' 'blackarch-exploitation' 'blackarch-webapp')
+url='https://github.com/vladko312/SSTImap'
+license=('GPL-3.0')
```
[the remark this answers is not in the document]
Comment (dualfade) [reply packages/sstimap/PKGBUILD:12]: Will update

File: packages/sstimap/PKGBUILD [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+pkgname=sstimap
+_pkgname=SSTImap
+pkgver=v1.2.r3.g0d45b44
+pkgrel=1
+pkgdesc='Automatic SSTI detection tool with interactive interface.'
+arch=('any')
+groups=('blackarch' 'blackarch-exploitation' 'blackarch-webapp')
+url='https://github.com/vladko312/SSTImap'
+license=('PL-3.0-or-later')
```
Comment (D3vil0p3r) [inline packages/sstimap/PKGBUILD:12]:
```suggestion
license=('GPL-3.0-or-later')
```

**Review by D3vil0p3r — CHANGES_REQUESTED** [review]

File: packages/sstimap/PKGBUILD (hunk 7 above)
Comment (dualfade) [reply packages/sstimap/PKGBUILD:12]: Muther of christ. Fixing.....

**Comment by dualfade** [comment]
Thanks all.

**Pull request merged**